### ISSUE TITLE
No more download progress?

### ISSUE THREAD

**Issue by LorenzoBettini** [eclipse-tycho/tycho#1890]
I noticed that Tycho 3.0.0 does not show download progress anymore in the log.
Is that due to the new transport layer? I haven't found any documentation on that

**Comment by laeubi** [comment]
Yes Tycho 3 do not print the progress but Tycho 4 has unified this with maven:
See https://github.com/eclipse-tycho/tycho/pull/1791

**Comment by LorenzoBettini** [comment]
Thanks. So, in case I want to see the log I'll have to wait for Tycho 4.0.0 right?

**Comment by laeubi** [comment]
You can also try to derive a patch for backporting:
https://github.com/eclipse-tycho/tycho/blob/master/CONTRIBUTING.md#backporting

**Comment by LorenzoBettini** [comment]
I can have a look at the backporting, but first I'd like to have a confirmation:

- can the log be restored by switching to the old transporter?
- can it be worthwhile to document such a change? Because one might have the feeling that something is not working without any logging they were used to

**Comment by laeubi** [comment]
If you think that printing random progress outweigh the improvements of the new transport (e.g. improved caching capabilities) you can enable the ECF transport.
Beside that there is no guarantee nor documentation about what/how Tycho do logging, if you still think it worth to mention somewhere you can add it to the migration guide/release notes.
